include/matx/operators/collapse.h

```diff
@@ -152,7 +152,7 @@ namespace matx
    *   Operator with collapsed input
    */
   template <int DIM, typename T1>
-    auto __MATX_INLINE__ lcollapse(const T1 &a)
+    auto __MATX_INLINE__ lcollapse(const T1 a)
```

Comment:
so that you can take an lvalue or an rvalue:   lcollapse( some_other_op_constructor);  I have been doing this to all ops as it is much less code/support than enabling perfect forwarding for both cases.  I'm counting on inline to save us here.